src/SFA.DAS.Support.Indexer.Worker/DependencyResolution/DefaultRegistry.cs

```diff
@@ -29,19 +29,21 @@ public DefaultRegistry()
                 x.GetInstance<IRequestContext>(),
                 x.GetInstance<ILoggingPropertyFactory>().GetProperties())).AlwaysUnique();
 
-
             var configuration = GetConfiguration();
             For<IWebConfiguration>().Use(configuration);
             For<ISearchSettings>().Use(configuration.ElasticSearch);
             For<ISiteSettings>().Use(configuration.Site);
-            For<ISiteConnectorSettings>().Use(configuration.SiteConnector);
+            For<ISiteConnectorSettings>().Use(configuration.SiteConnector ?? new SiteConnectorSettings());
```

Comment:
can this be removed now?

Reply:
leaving it there incase this get released without MI been setup yet.

Reply:
this branch and its config branch will only ever get merged at the same time so this isn't an option.

Reply:
removed